metastore/scripts/upgrade/hive/upgrade-4.0.0-beta-1-to-4.0.0-beta-2.hive.sql

```diff
@@ -1,3 +1,102 @@
 SELECT 'Upgrading MetaStore schema from 4.0.0-beta-1 to 4.0.0-beta-2';
 
 SELECT 'Finished upgrading MetaStore schema from 4.0.0-beta-1 to 4.0.0-beta-2';
+
+-- HIVE-27725
+DROP TABLE IF EXISTS `TAB_COL_STATS`;
```

Comment:
wow, that would drop the whole col statistics, can't we use something like `alter table drop column`?

Reply:
Hive table does not support drop column yet, we followed the previous behavior:
https://github.com/apache/hive/blob/cbc5d2d7d650f90882c5c4ad0026a94d2e586acb/metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-2-to-4.0.0-beta-1.hive.sql#L1-L8

Reply:
`COMPACTION_QUEUE` won't have side efects like stats removal;

Reply:
How about retaining the previous table structure? The code is compatible IMHO.